concordium-consensus/src/Concordium/MultiVersion.hs

```diff
@@ -837,7 +845,7 @@ shutdownMultiVersionRunner MultiVersionRunner{..} = mask_ $ do
     -- Acquire the write lock. This prevents further updates, as they will block.
     takeMVar mvWriteLock
     versions <- readIORef mvVersions
-    runLoggerT (forM_ versions $ \(EVersionedConfiguration vc) -> vcShutdown vc) mvLog
+    runLoggerT (forM_ @_ @_ @_ @() versions $ \(EVersionedConfiguration vc) -> vcShutdown vc) mvLog
```

Comment:
It's entirely unclear what type parameters these _ are specifying unless you find the order in an obscure place  in the GHC manual. Please provide all of them.

Reply:
With the refactorings and removal of the basic block state these are not necessary anymore.